### ISSUE TITLE
How to break foreach

### ISSUE THREAD

**Issue by Jobians** [tact-lang/tact#1066]
If I do, I get error 

```
foreach (key, value in self.oh_my) {
            // ...
           break;
        }
```

**Comment by novusnota** [comment]
@Jobians At the moment there is no `break` or `continue` in Tact language, at least not yet. And by extension, there are no such statements in FunC.

For now, consider using `return` instead — you have to separate that loop into its own function. Alternatively (but a bit more expensive), change the logic of your traversal, such as by adding a flag variable `let shouldProceed = true;`, then wrapping body of your `foreach` in an `if`-statement. Like this:

```tact
let shouldProceed = true;
foreach (key, value in self.oh_my) {
    if (shouldProceed) {
        // ... your code
        // and then instead of break, do:
        shouldProceed = false;
    }
}
```

**Comment by anton-trunov** [comment]
Will be resolved in #114

**Comment by anton-trunov** [comment]
@novusnota 

The solution with `if (shouldProceed) ` is not actually breaking anything, you will still have to traverse the whole map. While the solution with `return` does actually break
